package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ovio-site-gatsby",
+  "description": "Ovio website",
+  "version": "0.1.0",
+  "private": true,
+  "author": "Mamadou Ouologuem <mamadou.ouologuem02@gmail.com>",
+  "license": "GNU GENERAL PUBLIC LICENSE",
```

Comment:
Let's not add this until/if we open-source it - for now we're not necessarily using this license.

Reply:
Okay i will remove it 

I put it because Eric in is initial push, pushed the `Licence`